ci/setup_env.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+set -eEo pipefail
+
+export IS_A_RELEASE="false"
+export PROD_RELEASE="false"
+export DEV_RELEASE="false"
+export TEST_RELEASE="false"
+export COMMON_FILE_LOCATION='ci/common.sh'
+
+workdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"
+github_tag="${GITHUB_REF_NAME:-}"
+prod_release_branch="${PROD_RELEASE_BRANCH:-main}"
+prod_release_regex='^[0-9]+\.[0-9]+\.[0-9]+$'
+dev_release_regex='^[0-9]+\.[0-9]+\.[0-9]+(-[0-9]+)?(-rc[0-9]+){1}$'
+test_release_regex='^[0-9]+\.[0-9]+\.[0-9]+(-[0-9]+)?-.*$'
```

Comment:
Please modify this as:
`test_release_regex='^[0-9]+\.[0-9]+\.[0-9]+.+$'`.
Would make things easier and distinguish between `prod_release_regex` case.

Reply:
Don't we need `-` as well? like `d.d.d-ANYTHING` ?

Reply:
@JackPiri I would keep the format like N.N.N-Anything

Reply:
Ok it's fine to me.